File: HA6_2.py
def Digits(iNo):
    if (iNo == 1):
        print("One")
    elif (iNo == 2):
        print("Two")
    elif (iNo == 3):
        print("Three")
    elif (iNo == 4):
        print("Four")
    elif (iNo == 5):
        print("Five")
    elif (iNo == 6):
        print("Six")
    elif (iNo == 7):
        print("Seven")
    elif (iNo == 8):
        print("Eight")
    elif (iNo == 9):
        print("Nine")
    elif (iNo > 10):
        print("Invalid")
        
def main():
    iValue = 0
    print("Enter a Number")
    iValue = int(input())

    Digits(iValue)

if __name__=="__main__":
    main()

File: HA6_2.ts
import * as readline from 'readline'

function Pattern(iNo : number)
{
    if(iNo == 1)
    {
        console.log("One");
    }
    else if(iNo ==2)
    {
        console.log("Two");
    }
    else if(iNo == 3)
    {
        console.log("Three");
    }
    else if (iNo == 4)
    {
        console.log("Four");
    }
    else if(iNo == 5)
    {
        console.log("Five");
    }
    else if (iNo ==6)
    {
        console.log("Six");
    }
    else if (iNo == 7)
    {
        console.log("Seven");
    }
    else if(iNo == 8)
    {
        console.log("Eight");
    }
    else if(iNo == 9)
    {
        console.log("Nine");
    }
    else if (iNo >10)
    {
        console.log("Invalid");
    }
}

async function main()
{
    const rl = readline.createInterface({
        input: process.stdin,
        output: process.stdout
    })

    let iValue : number = 0

    iValue = parseInt(await askQuestion(rl, "Enter a number")) ||0

    Pattern (iValue)
}
function askQuestion(rl: readline.Interface, question: string): Promise<string>
{
    return new Promise((resolve) =>{
        rl.question(question,(answer : any) => {
            resolve(answer)
        })
    })
}
main()